CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## [v0.0.34] - 2023-09-27
+
+### New Features
+- feat: Add PlaygroundsSubgraphInspectorToolSpec to llama_hub (#535)
+- add full formatting and linting (#537, #538)
+
+### Smaller Features + Bug Fixes
+- fix: added missing leaders removed from a old PR (#540)
```

Comment:
- fix: added missing loaders removed from a old PR (#540)

I thought I fixed it on the PR